index.bs

```diff
@@ -532,6 +532,14 @@ that agent.  Agents should discard any authentication message whose
 `auth-initation-token` is set and does not match the `at` provided by the
 advertising agent.
 
+In the `psk-min-bits-of-entropy` field of the [=auth-capabilities=] messsage,
+agents may specify the minimum bits of entropy it requires for a PSK, in the
+range of 20 to 60 bits inclusive.  Agents must terminate any connection
```

Comment:
Why cap at 60? Isn't it pretty common for VPN type things for PSKs to have 128+ bits? Is there any reason other than performance to limit larger keys?

Reply:
Since we're (likely?) using memory hard functions, I might not have enough memory to do more than 60.  So if you say your minimum is 60bits, we're stuck.  Plus, in this context, 60bits is already overkill.